app/helpers/application_helper.rb

```diff
@@ -1,4 +1,18 @@
 # frozen_string_literal: true
 
 module ApplicationHelper
+  def content_structure
+    [
+      ['Image', 'simple_image'],
+      ['Book (ltr)', 'simple_book'],
+      ['Book (rtl)', 'simple_book_rtl'],
+      ['Document', 'document'],
+      ['File', 'file'],
+      ['Geo', 'geo'],
+      ['Media', 'media'],
+      ['3D', '3d'],
+      ['Map', 'maps'],
+      ['Webarchive seed', 'webarchive_seed']
+    ]
+  end
```

Comment:
moved to a helper from inline in the view so it is not duplicated.  useful since we are using a feature flag and need stimulus actions enabled for this menu but only when ocr is enabled via a feature flag.